lib/src/test/kotlin/tree/SearchTreeTest.kt

```diff
@@ -23,6 +23,35 @@ class SearchTreeTest {
         bst.set(10, "A")
     }
 
+    @Nested
+    inner class `Constructor tests` {
+        @Test
+        fun `tree without args`() {
+            val bst = BSTree<Int, String>()
+            assertEquals(0, bst.size)
+        }
+
+        @Test
+        fun `tree with one arg`() {
+            val bst = BSTree(1, "A")
+            assertEquals(listOf(Pair(1, "A")), bst.getEntities())
+        }
+
+        @Test
+        fun `tree with some args`() {
+            val bst: BSTree<Int, String> = BSTree(
+                arrayOf(
+                    Pair(1, "A"),
+                    Pair(2, "B"),
+                    Pair(3, "C"),
+                )
+            )
+
+            assertEquals(3, bst.size)
+            assertEquals(listOf(Pair(1, "A"), Pair(2, "B"), Pair(3, "C")), bst.getEntities())
```

Comment:
Это не нужно исправлять, но на будущее: пару можно очень легко задавать с помощью конструкции
```kotlin
listOf(1 to "A", 2 to "B", 3 to "C")
```
это буквально то же самое 

Если интересно - на самом деле ```to``` это **_инфиксная функция_**, которая на выходе дает пару

```suggestion
            assertEquals(listOf(1 to "A", 2 to "B", 3 to "C"), bst.getEntities())
```